sdk/maps/azure-maps-search/tests/recordings/test_client_e2e.test_fuzzy_search_address.yaml

```diff
@@ -0,0 +1,93 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      User-Agent:
+      - azsdk-python-maps-search/1.0-preview Python/3.9.13 (Windows-10-10.0.22000-SP0)
+      x-ms-client-id:
+      - <RealClientId>
+    method: GET
+    uri: https://atlas.microsoft.com/search/fuzzy/json?api-version=1.0&query=No.%20221%2C%20Sec.%202%2C%20Zhishan%20Rd.%2C%20Shilin%20Dist.%2C%20Taipei%20City%20111%2C%20Taiwan%20%28R.O.C.%29&subscription-key=pryWmPOH5EtRpxLMlGoMFni8b5rgPnMSR3ZFtrDj0hY
```

Comment:
This cloud be a severe exposure of credential. We should not commit subscription key in repo.